__mocks__/use-colors.ts

```diff
@@ -0,0 +1,34 @@
+export const mockUseColors = () => {
+  jest.mock("hooks/useColors", () => () => ({
```

Comment:
just curious - i noticed a lot of tests mock this hook. It seems like this hook just returns a static config map. What's the value in mocking it rather than just using the actual hook?